Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -2679,7 +2690,13 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
     /// - Returns: The best matching storage key if it was found, otherwise `nil`.
     public func identifier(forAssetName name: String, in parent: ResolvedTopicReference) -> String? {
         let bundleIdentifier = parent.bundleIdentifier
-        return assetManagers[bundleIdentifier]?.bestKey(forAssetName: name)
+        if let assetManager = assetManagers[bundleIdentifier] {
```

Comment:
Just for my understanding – Why is this extra check necessary? It looks like we register externally resolved assets with the asset manager above.

Reply:
`identifier(forAssetName:in:)` runs before `resolveAsset(named:bundleIdentifier:withType:)`. 

Only if we return a non-`nil` value here, does the above code get called to resolve the external asset.